hnn_core/cell.py

```diff
@@ -201,28 +208,23 @@ def dipole_insert(self, yscale):
                 # range variable 'dipole'
                 # set pointers to previous segment's voltage, with
                 # boundary condition
-                if i:
+                if i > 0:
```

Comment:
zen of Python: explicit is better than implicit. If `i` was `None` or `0` or `False` this would evaluate to `False` ... better to keep it clear.